src/SnowflakeTransformation.php

```diff
@@ -0,0 +1,257 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\SnowflakeTransformation;
+
+use Keboola\Component\Manifest\ManifestManager;
+use Keboola\Component\UserException;
+use Keboola\Datatype\Definition\Exception\InvalidTypeException;
+use Keboola\Datatype\Definition\GenericStorage as GenericDatatype;
+use Keboola\Datatype\Definition\Snowflake as SnowflakeDatatype;
+use Keboola\SnowflakeDbAdapter\Connection;
+use Keboola\SnowflakeDbAdapter\QueryBuilder;
+use Psr\Log\LoggerInterface;
+use Keboola\Component\Manifest\ManifestManager\Options\OutTableManifestOptions;
+
+class SnowflakeTransformation
+{
+    private const ABORT_TRANSFORMATION = 'ABORT_TRANSFORMATION';
+
+    private Connection $connection;
+
+    private LoggerInterface $logger;
+
+    private Config $config;
+
+    private array $databaseConfig;
+
+    public function __construct(Config $config, LoggerInterface $logger)
+    {
+        $this->logger = $logger;
+        $this->config = $config;
+        $this->databaseConfig = $config->getDatabaseConfig();
+        $this->connection = $this->createConnection();
+    }
+
+    public function createManifestMetadata(array $tables, ManifestManager $manifestManager): void
+    {
+        $getTables = $this->getTables($tables);
```

Comment:
divny nazvy, co treba

```suggestion
        $tableStructures = $this->getTables($tableNames);
```
nebo

```suggestion
        $tables = $this->getTables($tableNames);
```